updatecli/values.yaml

```
@@ -0,0 +1,6 @@
+github:
+  owner: olblak
```

```suggestion
  owner: civo
```

Should be updated to civo for Updatecli to submit changes to this repository.
If Updatecli is executed from a GitHub Action, then we could retrieve the owner and repository from the GitHub Action environment variable.